community/src/music.py

```diff
@@ -0,0 +1,8 @@
+import pyglet.media
+
+class MusicPlayer(pyglet.media.Player):
+    def __init__(self):
+        super().__init__()
+        
+        self.standby = False
+        self.next_time = 0
```

Comment:
This file uses spaces and is missing a newline at the end.

Could you run the formatter on it?

It's weird that the GitHub action isn't running on this branch...

Reply:
I thought my vscode was configured properly... I'll fix it when I get the chance